pkg/api/v1/types.go

```diff
@@ -2165,6 +2165,14 @@ type PodProxyOptions struct {
 	Path string `json:"path,omitempty"`
 }
 
+// ServiceProxyOptions is the query options to a Service's proxy call.
+type ServiceProxyOptions struct {
```

Comment:
why is this type different than the PodProxyOptions - they seem identical.  Is there ever going to be a /proxy sub-resource that has a different argument type?

@derekwaynecarr